tests/Integration/Behaviour/Features/Context/Domain/Product/ProductImageFeatureContext.php

```diff
@@ -39,7 +39,13 @@
 use PrestaShop\PrestaShop\Core\Domain\Product\Image\Exception\CannotRemoveCoverException;
 use PrestaShop\PrestaShop\Core\Domain\Product\Image\Query\GetProductImage;
 use PrestaShop\PrestaShop\Core\Domain\Product\Image\Query\GetProductImages;
+use PrestaShop\PrestaShop\Core\Domain\Product\Image\Query\GetShopProductImages;
 use PrestaShop\PrestaShop\Core\Domain\Product\Image\QueryResult\ProductImage;
+use PrestaShop\PrestaShop\Core\Domain\Product\Image\QueryResult\Shop\ProductImage as ProductImageInShop;
```

Comment:
Ah that's what I thought when I first saw the class name (even if it's in a sub namespace) it can "conflict" with the other `ProductImage` class (or at least be confusing) I was thinking it would deserve a dedicated different class name but I don't have a clear idea... `ShopProductImage` would be too similar with `ShopProductImages`

Maybe simply `ShopImageAssociation` or `ImageShopAssociation`?

Reply:
Done for ShopImageAssociation